pyproject.toml

```diff
@@ -23,11 +23,12 @@ dependencies = [
     "filelock",
     "fury",
     "indexed_gzip <= 1.8.7",
+    "Ingress2QSIRecon == 0.2.1",
     "jinja2 < 3.1",
     "matplotlib",
     "networkx ~= 2.8.8",
-    "nibabel <= 5.2.0",
-    "nilearn == 0.10.1",
+    "nibabel <= 6.0.0",
+    "nilearn",
```

Comment:
Is there a need to unpin this one?

Reply:
ah this was a holdout from some dependency differences between qsirecon and i2q, we should be able to pin it

Reply:
done in last commit